e2e/tests/hmctsAdministersCaseAfterSubmission_test.js

```diff
@@ -138,42 +139,12 @@ Scenario('HMCTS admin enters hearing details and submits', async (I, caseViewPag
 });
 
 Scenario('HMCTS admin creates multiple orders for the case', async (I, caseViewPage, createOrderEventPage) => {
-  await caseViewPage.goToNewActions(config.administrationActions.createOrder);
-  await createOrderEventPage.selectType(orders[0].type);
-  await I.retryUntilExists(() => I.click('Continue'), '#order_title');
-  await createOrderEventPage.enterC21OrderDetails();
-  await I.retryUntilExists(() => I.click('Continue'), '#judgeAndLegalAdvisor_judgeTitle');
-  await createOrderEventPage.enterJudgeAndLegalAdvisor(orders[0].judgeAndLegalAdvisor.judgeLastName, orders[0].judgeAndLegalAdvisor.legalAdvisorName);
-  await I.completeEvent('Save and continue');
-  let orderTime = new Date();
-
-  I.seeEventSubmissionConfirmation(config.administrationActions.createOrder);
-  caseViewPage.selectTab(caseViewPage.tabs.orders);
-  I.seeAnswerInTab(1, 'Order 1', 'Type of order', orders[0].type);
-  I.seeAnswerInTab(2, 'Order 1', 'Order title', orders[0].title);
-  I.seeAnswerInTab(4, 'Order 1', 'Order document', orders[0].document);
-  I.seeAnswerInTab(5, 'Order 1', 'Date and time of upload', dateFormat(orderTime, 'd mmmm yyyy'));
-  I.seeAnswerInTab(1, 'Judge and legal advisor', 'Judge or magistrate\'s title', orders[0].judgeAndLegalAdvisor.judgeTitle);
-  I.seeAnswerInTab(2, 'Judge and legal advisor', 'Last name', orders[0].judgeAndLegalAdvisor.judgeLastName);
-  I.seeAnswerInTab(3, 'Judge and legal advisor', 'Legal advisor\'s full name',  orders[0].judgeAndLegalAdvisor.legalAdvisorName);
-
-  await caseViewPage.goToNewActions(config.administrationActions.createOrder);
-  await createOrderEventPage.selectType(orders[1].type);
-  await I.retryUntilExists(() => I.click('Continue'), '#judgeAndLegalAdvisor_judgeTitle');
-  await createOrderEventPage.enterJudgeAndLegalAdvisor(orders[1].judgeAndLegalAdvisor.judgeLastName, orders[1].judgeAndLegalAdvisor.legalAdvisorName);
-  await I.retryUntilExists(() => I.click('Continue'), '#orderFurtherDirections_directionsNeeded');
-  await createOrderEventPage.enterDirections('example directions');
-  await I.completeEvent('Save and continue');
-  orderTime = new Date();
-
-  I.seeEventSubmissionConfirmation(config.administrationActions.createOrder);
-  caseViewPage.selectTab(caseViewPage.tabs.orders);
-  I.seeAnswerInTab(1, 'Order 2', 'Type of order', orders[1].type);
-  I.seeAnswerInTab(2, 'Order 2', 'Order document', orders[1].document);
-  I.seeAnswerInTab(3, 'Order 2', 'Date and time of upload', dateFormat(orderTime, 'd mmmm yyyy'));
-  I.seeAnswerInTab(1, 'Judge and legal advisor', 'Judge or magistrate\'s title', orders[1].judgeAndLegalAdvisor.judgeTitle);
-  I.seeAnswerInTab(2, 'Judge and legal advisor', 'Last name', orders[1].judgeAndLegalAdvisor.judgeLastName);
-  I.seeAnswerInTab(3, 'Judge and legal advisor', 'Legal advisor\'s full name',  orders[1].judgeAndLegalAdvisor.legalAdvisorName);
+  for(let i = 0; i < orders.length; i++) {
```

Comment:
with this construct, it would be hard to determine which order failed.

Reply:
Valid, was trying to reduce repetition, would you value the repetition more in this situation?

Reply:
Discussed offline